ralbot/html/exporter.py

```diff
@@ -82,12 +83,17 @@ def export(self, node, output_dir, **kwargs):
             parent home page.
         """
 
+        # if not a list
+        if not isinstance(nodes, list):
+            nodes = [nodes]
+
         # If it is the root node, skip to top addrmap
-        if isinstance(node, RootNode):
-            node = node.top
+        for i, node in enumerate(nodes):
+            if isinstance(node, RootNode):
+                nodes[i] = node.top
 
         self.footer = kwargs.pop("footer", "Generated by RALBot HTML")
-        self.title = kwargs.pop("title", "%s Reference" % node.get_property("name"))
+        self.title = kwargs.pop("title", "Reference")
```

Comment:
Rather than discarding the root name, would prefer that the default title behavior remained as it did before.
Maybe do the following instead:
```suggestion
        self.title = kwargs.pop("title", "%s Reference" % nodes[0].get_property("name"))
```